src/lib.rs

```diff
@@ -13,27 +13,25 @@
 //! This repository has been created so there's a unique library that holds the
 //! types and functions required to perform keys operations.
 
-#![cfg_attr(not(feature = "std"), no_std)]
+#![no_std]
 #![deny(missing_docs)]
 #![allow(non_snake_case)]
 
-pub use decode::decode as jubjub_decode;
-
-/// PKI Errors
-pub use errors::Error;
+/// Public Key
+pub use keys::public::PublicKey;
+/// Secret Key
+pub use keys::secret::SecretKey;
 /// Public Spend Key
-pub use spend::public::PublicKey as PublicSpendKey;
+pub use keys::spend::public::PublicSpendKey;
 /// Secret Spend Key
-pub use spend::secret::SecretKey as SecretSpendKey;
+pub use keys::spend::secret::SecretSpendKey;
 /// Stealth Address
-pub use spend::stealth::{Ownable, StealthAddress};
+pub use keys::spend::stealth::{Ownable, StealthAddress};
 /// ViewKey
 pub use view::ViewKey;
```

Comment:
Isn't the view key just a duplicate of the `SecretKey`? In practice, this struct is duplicated

Reply:
No, it's not. They have different fields.